.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: true
 contact_links:
   - name: Jetty Security Reports
-    url: https://eclipse.dev/jetty/security_reports.php
+    url: https://jetty.org/security_reports.php
```

Comment:
This URL doesn't exist.
We should create one and use it here.

Reply:
done to https://jetty.org/security.html